src/utils/transform.js

```diff
@@ -558,16 +567,26 @@ export const getDependentQueries = (store, componentID, orderOfQueries = []) =>
 		const customQuery = store.customQueries[component];
 		if (!isInternalComponent(component)) {
 			const calcValues = store.selectedValues[component] || store.internalValues[component];
-			// Only include queries for that component that has `customQuery` or `value` defined
-			if (((calcValues && calcValues.value) || customQuery) && !finalQuery[component]) {
+			const imageValue = calcValues && calcValues.meta && calcValues.meta.imageValue;
+			// Only include queries for that component that has `customQuery` or `value` or
+			// `imageValue` incase of searchbox defined
+			if (((calcValues && (calcValues.value || imageValue)) || customQuery)
+				&& !finalQuery[component]) {
 				let execute = false;
 				const componentProps = store.props[component];
 				if (
 					Array.isArray(orderOfQueries)
 					&& orderOfQueries.includes(component)
 					&& !(
 						componentProps.componentType === componentTypes.searchBox
-						&& componentProps.enableAI
+						/**
+						 * We want to fire a search query,
+						 * 	when enableAI
+						 * 	OR
+						 * 	when autosuggest is false
+						 */
+						&& (componentProps.enableAI
+						|| componentProps.autosuggest === false)
```

Comment:
@SavvyShah I wonder were we not doing it already until now for `autosuggest` set to false?

Reply:
No. We were actually making a suggestion query instead of a search query.